### PR TITLE
Create SHACL graph for Project Page on boarding forms

### PR DESCRIPTION
**Pull request by blushi**
closes: https://github.com/regen-network/regen-registry/issues/484

### CODE REVIEW

File: schema/project-page.ttl [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+  sh:datatype xsd:boolean ;
+  sh:hasValue true ;
+.
+
+regen:ProjectQuoteShape
+  a rdfs:Class, sh:NodeShape ;
+  sh:property [
+    sh:path regen:quote ;
+    sh:name "Quote from land steward or other important stakeholder" ;
+    sh:description "Choose an inspiring quote that helps others understand why this project is important. <a href=\"\">See an example»</a>" ;
```
Comment (mhagel) [inline schema/project-page.ttl:387]: Where will `description` be seen? As part of a validation message?

**Review by mhagel — APPROVED** [review]
It would be helpful to have brief description in the PR about what this code implements. Maybe in plain English, like "Sets the structure for data organization when updating project page." Or anything you think might be helpful for somebody who is new to SHACL, like me :) 

Looks good!

**Comment by blushi** [comment]
> It would be helpful to have brief description in the PR about what this code implements. Maybe in plain English, like "Sets the structure for data organization when updating project page." Or anything you think might be helpful for somebody who is new to SHACL, like me :)
> 
> Looks good!

It's already stated in the title, it implements the Project Page on boarding forms validation. How to be more specific? Or do you mean add more comments in the code?

File: schema/project-page.ttl (hunk 1 above)
Comment (blushi) [reply schema/project-page.ttl:387]: If you look at SHACL docs https://www.w3.org/TR/shacl/#nonValidation, you can see than `sh:description` is not being used for validation but could be used down the road by the form generation front-end library.

**Comment by mhagel** [comment]
> > It would be helpful to have brief description in the PR about what this code implements. Maybe in plain English, like "Sets the structure for data organization when updating project page." Or anything you think might be helpful for somebody who is new to SHACL, like me :)
> > Looks good!
> 
> It's already stated in the title, it implements the Project Page on boarding forms validation. How to be more specific? Or do you mean add more comments in the code?

Yes the title description is helpful.  Especially now that I have been building the front end for this. It just wasn't obvious to me at first.

File: schema/project-page.ttl [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+    ]
+  ) ;
+  # The following property isn't actually used for validation but could be used for
+  # form generation later on.
+  # Indeed, we have a separate form in the UI for choosing which entities to display
+  # on the project page, but we're actually storing this info as part of regen:landOwner,
+  # regen:landSteward and regen:projectDeveloper along with the "Roles" info.
+  sh:property [
+    sh:name "Choose the entities to show on the project page:" ;
+    sh:description "Showing more entities increases the salability of the project. You must show at least one entity on the project page. These entities can only be edited in the previous step. <a href=\"\">See an example»</a>" ;
```
Comment (mhagel) [inline schema/project-page.ttl:107]:
```suggestion
    sh:description "Showing more entities increases the salability of the project. You must show at least one entity on the project page. These entities can only be edited in the previous step." ;
```

In the UI, these "See an example" links will open a modal, rather than link to somewhere else. I think it may be best to leave them off the descriptions, and the front end will add them in to handle the popup logic. WDYT?

Comment (mhagel) [reply schema/project-page.ttl:107]: https://www.figma.com/file/a1A8H2i3gBzAq915IbfDAL/Onboarding-Flow?node-id=6539%3A225 for popup reference

Comment (blushi) [reply schema/project-page.ttl:107]: Yeah makes sense

**Review by haveanicedavid — APPROVED** [review]
